src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Prefer using 'const' over 'let' for variables that do not change. In this case, 'price' is not reassigned after its initial declaration, so it should be declared with 'const'.